docs/assets/sass/docs.scss

```diff
@@ -1,4 +1,4 @@
-@import "node_modules/govuk-frontend/all";
+@import "node_modules/govuk-frontend/govuk/all";
```

Comment:
Mat has suggested if we added a new feature to the prototype kit extensions config, we could simplify these kinds of imports to "govuk/all"

https://github.com/alphagov/govuk-design-system-architecture/pull/15#issuecomment-500786673